cabal.project

```diff
@@ -21,7 +21,7 @@ allow-newer: *:Cabal, *:array, *:base, *:binary, *:^bytestring, *:containers,
   *:aeson, *:th-abstraction
 
 repository head.hackage.ghc.haskell.org
-   url: https://ghc.gitlab.haskell.org/head.hackage/
+   url: http://ghc.gitlab.haskell.org/head.hackage/
    secure: True
```

Comment:
`secure: True` :eyes:

Reply:
woops, that was me

Reply:
`secure:` is about [Hackage package(index) signatures](https://github.com/haskell/hackage-security) based on the keys listed below.
It's separate from transport security.

Reply:
Yes :). For the same reason APT repositories can be hosted on HTTP. I just thought it looked funny.

What's the reason for changing it to HTTP? It seems the HTTPS version works just fine.

Reply:
Turns out cabal-install can do http internally, but not https.
And when curl is installed it will automagically use that to do http(s).